src/amo/components/Collection/index.js

```diff
@@ -136,7 +136,7 @@ export class CollectionBase extends React.Component<Props> {
     if (collection && addonsPageChanged) {
       this.props.dispatch(fetchCurrentCollectionPage({
         errorHandlerId: errorHandler.id,
-        page: location.query.page,
+        page: location.query.page || 1,
```

Comment:
Would it make sense to default `page` to 1 in [`fetchCurrentCollectionPage`](https://github.com/mozilla/addons-frontend/blob/master/src/amo/reducers/collections.js#L215) instead of dealing with this in the component?

Reply:
I thought about it too, but since `fetchCurrentCollectionPage()` implies to fetch a specific page, I thought it would be better to always require a `page`.